processor/src/tests/crypto_ops.rs

```diff
@@ -0,0 +1,150 @@
+use super::{super::execute, build_inputs, compile};
+use super::{build_stack_state, Felt, FieldElement};
+use crate::Word;
```

Comment:
nit: could these be combine into a single `use super::...`? For `Word` import you'd need to import it in the super module.